central/src/gitlab.rs

```diff
@@ -24,6 +24,28 @@ pub struct GitLabProjectAccessTokenProvider {
     client: reqwest::Client,
 }
 
+/// Derive the bridgehead configuration repository from the beam id of the requester.
+/// Example return value: "bridgehead-configurations/bridgehead-config-berlin"
+fn derive_bridgehead_config_repo_from_beam_id(requester: &AppId) -> Result<String, String> {
+    let mut parts = requester.as_ref().splitn(3, '.');
+    let (_app, proxy, broker) = (parts.next().unwrap(), parts.next().unwrap(), parts.next().unwrap());
```

Comment:
@timakro Sorry for being late, but do you have to unwrap? Isn't this available as a function of the AppId?

Reply:
`AppId` is missing a function to get the broker. Should I open a PR in the beam repo to add this?